### ISSUE TITLE
Editable hero image

### ISSUE THREAD

**Issue by shouston3** [LDMW/app#370]
See #321 

Depends on #349

- [ ] Create field in wagtail to upload a hero image ([image chooser field](http://docs.wagtail.io/en/v1.9/editor_manual/new_pages/inserting_images.html#inserting-images-using-the-image-chooser-field) `classname="col6"`)
- [ ] Add field for image positioning (ideally a dropdown with `center`, `left`, `right`, `top`, `bottom`) (`classname="col6"`)

**Comment by iteles** [comment]
@shouston3 Is this not a duplicate of https://github.com/LDMW/app/issues/321 ?

Please add the technical detail as a comment to that issue, no need to duplicate them 👍

**Comment by shouston3** [comment]
Agreed, should have put this info in the pr description instead

Duplicate of #321, closing
